docs/index.md

```diff
@@ -1,21 +1,51 @@
 ---
-page_title: "firehydrant Provider"
-subcategory: ""
-description: |-
-
+page_title: "Provider: FireHydrant"
 ---
 
 # FireHydrant Provider
 
-Welcome to the FireHydrant Terraform provider! With this provider you can create and manage resources on your [FireHydrant](https://www.firehydrant.io) organization such as incident runbooks, services, teams, and more!
-
-
-
-## Schema
-
-### Required
-
-- **api_key** (String, Optional) This is your API key (typically a bot token in FireHydrant) that is used to manage resources in FireHydrant. If set, the environment variable `FIREHYDRANT_API_KEY` will be used.
-
-### Optional
-- **firehydrant_base_url** (String, Optional)
+Welcome to the FireHydrant Terraform provider! With this provider you can create
+and manage resources on your [FireHydrant](https://www.firehydrant.io) organization
```

Comment:
Should this be `https://www.firehydrant.com` we redirect so I don't think it's a big deal

Reply:
oh yeah, it definitely should. good catch!